test/controller/villains.test.js

```diff
@@ -1,18 +1,59 @@
 const chai = require('chai')
 const sinon = require('sinon')
 const sinonChai = require('sinon-chai')
-const { describe, it } = require('mocha')
+const {
```

Comment:
you are missing after in your require from mocha as well as in your describe controllers villains portion of declaring